Tests/MountebankSwiftTests/ExampleData/Stub/Response/Behavoir+Examples.swift

```diff
@@ -23,11 +23,11 @@ extension Behavior {
         )
 
         static let copy = Example(
-            value: Behavior.copy([
-                "from": "path",
-                "into": "${code}",
-                "using": ["method": "regex", "selector": "\\d+"],
-            ]),
+            value: Behavior.copy(Copy(
+                from: "path",
+                into: "${code}",
+                using: Behavior.Copy.Using(method: .regex, selector: "\\d+")
+            )),
```

Comment:
Add more examples here so we've got test coverage for en/decoding of these new objects? Perhaps add all the examples from https://www.mbtest.org/docs/api/behaviors#behavior-copy here?

Reply:
I've added all examples see Tests/MountebankSwiftTests/ExampleData/Stub/Response/Behavoir+Examples.swift